### DISCUSSION TITLE
[QA] InternLM2微调时数据的max_token

### DISCUSSION THREAD

**Discussion by MING-ZCH** [InternLM/InternLM#687] [Q&A] [answered]
### Describe the question.

我有长文本数据准备全参微调InternLM2-7b-chat，想询问支持的数据的max_length长度？
Thanks！

**Comment by ZwwWayne** [comment, marked as answer]
仅供参考：出于效率和效果的 trade-off，我们目前用32K长文本进行了训练，然后外推到200K。

**Reply by MING-ZCH** [reply]
感谢！
